source/carbonstorage.rst

```diff
@@ -178,11 +175,10 @@ Land use/land cover
 
 Global land use data is available from:
 
- *  NASA: https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd12q1 (MODIS multi-year global landcover data provided in several classifications)
- *  The European Space Agency: https://www.esa-landcover-cci.org (Three global maps for the 2000, 2005 and 2010 epochs) 
- *  The University of Maryland�s Global Land Cover Facility: http://glcf.umd.edu/data/landcover/ (data available in 1 degree, 8km and 1km resolutions).
```

Comment:
Is there a reason this link was deleted?

Reply:
The site no longer exists, and it seems the whole project was discontinued. I couldn't find the same dataset anywhere.